importer/RsSr.py

```diff
@@ -0,0 +1,129 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class RsSr(Monument):
+
+    def set_address(self):
+        """
+        Set address / directions.
+
+        Address, with attached city, if there's digits in it.
+        Otherwise directions.
+        """
+        if self.has_non_empty_attribute("address"):
+            if self.has_non_empty_attribute("city"):
```

Comment:
Do we not want to add directions if there is no city?

Reply:
Added.